File: Beginner/A - Books (BIT2A)/books.py
t = int(input())


while t:
    N = int(input())
    A = list(map(int, input().split()))

    c = 0
    for i in range(N):
        for j in range(i+1, N):
            if A[i]<A[j]:
                c = c+1
        print(c, end=' ')
        c = 0
    
    t = t-1

File: Beginner/A - Books (BIT2A)/books.dart
import 'dart:convert';
import 'dart:io';

void main() {
  int t = int.parse(stdin.readLineSync(encoding: utf8)!);

  while (t > 0) {
    int N = int.parse(stdin.readLineSync(encoding: utf8)!);
    var A = stdin
        .readLineSync(encoding: utf8)!
        .split(" ")
        .map((e) => int.parse(e))
        .toList();

    int c = 0;
    for (int i = 0; i < N; i++) {
      for (int j = i + 1; j < N; j++) {
        if (A[i] < A[j]) {
          c++;
        }
      }

      stdout.write("$c ");
      c = 0;
    }

    t--;
  }
}
